src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -979,6 +979,35 @@ public static void Clear()
             Assert.Same(string.Empty, builder.ToString());
         }
 
+        [Fact]
+        public static void Clear_ClearEmptyStringBuilder_CapacityNotZero()
+        {
+            var builder = new StringBuilder();
+            builder.Clear();
+            Assert.NotEqual(0, builder.Capacity);
+        }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(10000)]
+        public static void Clear_AppendAndInsertBeforeClearManyTimes_CapacityStaysWithinRange(int times)
```

Comment:
You will need to disable this for .NET Framework

Reply:
With a note of the issue # fixing it, and that on NETFX you end up with 100042

Reply:
Suggestion: I would also add cases for the other possible outcomes of Clear (where we don't end up with ~1.2*Length of the original string)